js/private/coverage/merger.bzl

```diff
@@ -16,17 +16,23 @@ _ATTRS = {
 # Do the opposite of _to_manifest_path in
 # https://github.com/bazelbuild/rules_nodejs/blob/8b5d27400db51e7027fe95ae413eeabea4856f8e/nodejs/toolchain.bzl#L50
 # to get back to the short_path.
-# TODO(2.0): fix toolchain so we don't have to do this
-def _target_tool_path_to_short_path(tool_path):
+# TODO(3.0): remove this after a grace period for the DEPRECATED toolchain attributes
```

Comment:
Should this be 2.0?

Reply:
Nope. I'm punting removing support for rules_nodejs < 6.1.0 to a future 3.0 release